### PR TITLE
Update various images

### PR DESCRIPTION
**Pull request by nusE0726844**
Add output images for the Appointment commands.

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1634,10 +1634,7 @@ vaccination add Dose 1 (Moderna)
 
 #### Adding a valid appointment
 
-``` text
-appointment add --p 1 --s 2024-01-01 1330 --e 2024-01-01 1400 /
-    --v Dose 1 (Moderna)
-```
+appointment add --p 1 --s 2024-01-01 1330 --e 2024-01-01 1400 --v Dose 1 (Moderna)
```
Comment (francisyzy) [inline docs/DeveloperGuide.md:1637]: I think can still warp this around ``

ie:
`appointment add --p 1 --s 2024-01-01 1330 --e 2024-01-01 1400 --v Dose 1 (Moderna)`

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -1723,7 +1720,7 @@ appointment add --p 1 --s 2024-01-01 1330 --e 2024-01-01 1400 /
 
 ##### Expected
 
-![Expected display](images/appointment/dg/AddAppointmentValid.png)
+![Valid Add Appointment](images%2Fappointment%2Fdg%2FAddAppointmentValid.PNG)
```
Comment (francisyzy) [inline docs/DeveloperGuide.md:1723]: Hmm why is it not using the `/` but `%2F`

Comment (nusE0726844) [reply docs/DeveloperGuide.md:1723]: The formatting changed the `/` to `%2F` when pasting

Comment (francisyzy) [reply docs/DeveloperGuide.md:1723]: Will have to check if it will cause any issue when building

**Review by francisyzy — APPROVED** [review]

**Pull request merged**